cmd/context.go

```diff
@@ -109,6 +109,28 @@ var clearContextCmd = &cobra.Command{
 	},
 }
 
+var loginContextCmd = &cobra.Command{
```

Comment:
I guess this is very Qlik Sense Enterprise for Windows specific functionality, so only `login` might be a bit confusing.

Reply:
Let us keep login